src/services/Peers.js

```diff
@@ -18,10 +19,39 @@ module.exports = {
     })
   },
 
+  formatUptimes: function(peers) {
+    function format(time) {
+      if (!time) return {name: '', time: -1};
+      var seconds = time;
+      var minutes = Math.floor(time / 60);
+      seconds -= minutes * 60;
+      var hours = Math.floor(minutes / 60);
+      minutes -= hours * 60;
+      var days = Math.floor(hours / 24);
+      hours -= days * 24;
+      var dayName = days + ' ' + 'Day' + (days > 1 ? 's' : '');
+      var hourName = hours + ' ' + 'Hour' + (hours > 1 ? 's' : '');
+      var minName = minutes + ' ' + 'min';
+      var secName = seconds + ' ' + 'sec';
```

Comment:
minutes and seconds are long words, so I thought it's a good idea to use
Day(s), Hour(s), min, sec.